UI/installer/mp-installer.nsi

```diff
@@ -103,31 +118,38 @@ Function PreReqCheck
 		gotPatch:
 	${EndIf}
 
-	; 32 bit Visual Studio 2013 runtime check
+!ifdef INSTALL64
+	; 64 bit Visual Studio 2017 runtime check
 	ClearErrors
-	GetDLLVersion "MSVCR120.DLL" $R0 $R1
-	GetDLLVersion "MSVCP120.DLL" $R0 $R1
-	IfErrors vs2013Missing vs2013OK1
-	vs2013Missing:
-		MessageBox MB_YESNO|MB_ICONEXCLAMATION "Your system is missing runtime components that ${APPNAME} requires. Please make sure to install both vcredist_x64 and vcredist_x86. Would you like to download them?" IDYES vs2013true IDNO vs2013false
-		vs2013true:
-			ExecShell "open" "https://obsproject.com/visual-studio-2013-runtimes"
-		vs2013false:
+	SetOutPath "$TEMP\OBS"
+	File check_for_64bit_visual_studio_2017_runtimes.exe
+	ExecWait "$TEMP\OBS\check_for_64bit_visual_studio_2017_runtimes.exe" $R0
+	Delete "$TEMP\OBS\check_for_64bit_visual_studio_2017_runtimes.exe"
+	RMDir "$TEMP\OBS"
+	IntCmp $R0 126 vs2017Missing_64 vs2017OK_64
+	vs2017Missing_64:
+		MessageBox MB_YESNO|MB_ICONEXCLAMATION "Your system is missing runtime components that ${APPNAME} requires.  Would you like to download them?" IDYES vs2017true_64 IDNO vs2017false_64
+		vs2017true_64:
+			ExecShell "open" "https://obsproject.com/visual-studio-2017-runtimes-64-bit"
+		vs2017false_64:
 		Quit
```

Comment:
Out of curiosity, would it be possible to change this part of the script to instead show a "Retry" button, rather than immediately closing when the user clicks either button? Taking them out of the flow seems counter-intuitive.

Reply:
I think it's possible, but in the interest of just getting things up to current standards, going to defer this to a future update to the installer script.